dhcpv6/option_fqdn.go

```diff
@@ -0,0 +1,27 @@
+package dhcpv6
+
+import "fmt"
+
+// OptFQDN implements OptionFQDN option.
+//
+// https://tools.ietf.org/html/rfc4704
+type OptFQDN string
+
+// Code returns the option code.
+func (op OptFQDN) Code() OptionCode {
+	return OptionFQDN
+}
+
+// ToBytes serializes the option and returns it as a sequence of bytes
+func (op OptFQDN) ToBytes() []byte {
+	return []byte(op)
+}
+
+func (op OptFQDN) String() string {
+	return fmt.Sprintf("OptFQDN{%s}", string(op))
+}
+
+// ParseOptFQDN deserializes from bytes to build a OptFQDN structure.
+func ParseOptFQDN(data []byte) (OptFQDN, error) {
+	return OptFQDN(string(data)), nil
```

Comment:
What about the flags specified in the `rfc4704`?

Reply:
This is a copy of original internal work, which missed flags since it simply need to return hostname. Just add it. 

I am not super familiar with option impl here, try to mimic the way how other options are implemented, let me know and happy to learn